app/io/pathfinder/models/Commodity.java

```diff
@@ -0,0 +1,35 @@
+package io.pathfinder.models;
+
+import com.avaje.ebean.Model;
+import javax.persistence.*;
```

Comment:
Please don't use wildcard imports, it makes it harder for other people to figure out what the code is doing.